### PR TITLE
#1424 redesign label edit form

### PR DESCRIPTION
**Pull request by ZaimBeshich1**

### CODE REVIEW

File: packages/ketcher-react/src/script/ui/dialog/toolbox/labeledit.jsx [hunk 1]
```diff
@@ -66,14 +67,28 @@ function deserialize(value) {
...
+  }
   return (
     <Dialog
       title="Label Edit"
-      className="labeledit"
       valid={() => valid}
+      withDivider={true}
       result={() => deserialize(result.label)}
+      className={styles.labelEdit}
+      buttons={getButton()}
```
Comment (AnaMoskvina) [inline packages/ketcher-react/src/script/ui/dialog/toolbox/labeledit.jsx:91]: basically, Apply button is OK button
we can add following props:
 buttons: ['OK'],
 buttonsNameMap={{ OK: 'Apply' }}

in this case, no other changes are needed
You can check out TemplateDialog as a reference

File: packages/ketcher-react/src/script/ui/dialog/toolbox/labelEdit.module.less [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    input {
+      margin-left: 8px;
+      min-width: 140px;
+    }
+  }
+
+  footer {
+    padding: 16px 12px 12px;
```
Comment (AnaMoskvina) [inline packages/ketcher-react/src/script/ui/dialog/toolbox/labelEdit.module.less:42]: I will add these changes to global styles, so you can remove them

File: packages/ketcher-react/src/script/ui/dialog/toolbox/labelEdit.module.less [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    .okButton {
```
Comment (AnaMoskvina) [inline packages/ketcher-react/src/script/ui/dialog/toolbox/labelEdit.module.less:44]: with buttonsNameMap prop we won't need it
also, you can use corresponding mixins when needed

**Review by AnaMoskvina — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/ui/dialog/toolbox/labeledit.jsx (hunk 1 above)
Comment (ZaimBeshich1) [reply packages/ketcher-react/src/script/ui/dialog/toolbox/labeledit.jsx]: Ok thank u, now I understand how to use it.

File: packages/ketcher-react/src/script/ui/dialog/toolbox/labelEdit.module.less (hunk 2 above)
Comment (ZaimBeshich1) [reply packages/ketcher-react/src/script/ui/dialog/toolbox/labelEdit.module.less]: deleted

File: packages/ketcher-react/src/script/ui/dialog/toolbox/labelEdit.module.less (hunk 3 above)
Comment (ZaimBeshich1) [reply packages/ketcher-react/src/script/ui/dialog/toolbox/labelEdit.module.less]: deleted

File: packages/ketcher-react/src/script/ui/dialog/toolbox/labeledit.jsx [hunk 4]
```diff
@@ -66,14 +67,28 @@ function deserialize(value) {
 
 function LabelEdit(props) {
   const init = { label: props.letter || serialize(props) }
-  const { formState, ...prop } = props
+  const { formState, onOk, ...prop } = props
```
Comment (AnaMoskvina) [inline packages/ketcher-react/src/script/ui/dialog/toolbox/labeledit.jsx:70]: as we extract onOk prop here, it won't get into params prop in Dialog
it should affect functionality

Comment (ZaimBeshich1) [reply packages/ketcher-react/src/script/ui/dialog/toolbox/labeledit.jsx]: Deleted unused prop

**Review by AnaMoskvina — APPROVED** [review]

**Review by AndreiMazol — APPROVED** [review]

**Pull request merged**